doc/lisa_tests.rst

```diff
@@ -12,9 +12,9 @@ From the root of LISA, you can run those tests like so:
 
 >>> python3 -m nose
 >>> # You can also target specific test modules
->>> python3 -m nose --tests=lisa/tests/lisa/test_test_bundle.py
+>>> python3 -m nose tests/test_test_bundle.py
```

Comment:
is it working without --tests now ? It used to not work.

Reply:
Yeah, because we had to specify it in the config file and I think it would always use it, so we'd be forced to override it. It works now.